astropy/io/fits/fitstime.py

```diff
@@ -0,0 +1,251 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import re
+import warnings
+from collections import defaultdict, OrderedDict
+
+import numpy as np
+
+from . import Header, Card
+
+from ...coordinates import EarthLocation
+from ...table import Column
+from ...time import Time
```

Comment:
@saimn So from what I understood, we would want to reduce the probability of having circular import issues. By relative imports I'm assuming you mean the ones within io.fits. So should I keep `from . import Header, Card` as a global import and all the other sub-package imports (`from ...coordinates import EarthLocation `, etc.) within FITS_time() ?